templates/zerver/register.html

```diff
@@ -23,61 +23,108 @@ <h1>You&rsquo;re almost there.</h1>
-                              class="{% if root_domain_landing_page %}required{% endif %} subdomain" type="text"
-                              placeholder="acme"
-                              value="{% if form.realm_subdomain.value() %}{{ form.realm_subdomain.value() }}{% endif %}"
-                              name="realm_subdomain" maxlength={{ MAX_REALM_SUBDOMAIN_LENGTH }}
-                              {% if root_domain_landing_page %}required{% endif %} />
-                            <label for="id_team_subdomain" class="realm_subdomain_label">.{{ external_host }}</label>
-                            <p id="id_team_subdomain_error_client" class="error help-inline text-error"></p>
+                        <div id="subdomain_section" {% if root_domain_available and
+                          not form.realm_subdomain.errors and not form.realm_subdomain.value() %}style="display: none;"{% endif %}>
+                            <div class="or"><span>{{ _('OR') }}</span></div>
+                            <div class="inline-block relative">
+                                <input id="id_team_subdomain"
+                                  class="{% if root_domain_landing_page %}required{% endif %} subdomain" type="text"
+                                  placeholder="acme"
+                                  value="{% if form.realm_subdomain.value() %}{{ form.realm_subdomain.value() }}{% endif %}"
+                                  name="realm_subdomain" maxlength={{ MAX_REALM_SUBDOMAIN_LENGTH }}
+                                  {% if root_domain_landing_page %}required{% endif %} />
+                                <label for="id_team_subdomain" class="realm_subdomain_label">.{{ external_host }}</label>
+                                <p id="id_team_subdomain_error_client" class="error help-inline text-error"></p>
+                            </div>
+                            {% if form.realm_subdomain.errors %}
+                                {% for error in form.realm_subdomain.errors %}
+                                <p class="error help-inline text-error team_subdomain_error_server">{{ error }}</p>
+                                {% endfor %}
+                            {% endif %}
+                        </div>
+                        <div class="help-box margin-top">
+                            {{ _("The address you'll use to log in to your organization.") }}
                         </div>
-                        {% if form.realm_subdomain.errors %}
-                            {% for error in form.realm_subdomain.errors %}
-                            <p class="error help-inline text-error team_subdomain_error_server">{{ error }}</p>
-                            {% endfor %}
-                        {% endif %}
                     </div>
-                    <div class="help-box margin-top">
-                        {{ _("The address you'll use to log in to your organization.") }}
```

Comment:
In this whole above chunk of the html file, are we changing anything meaningful? Or just indentation? I'm asking because github's display of html file edits is super confusing sometimes.

Reply:
There is no change in this chunk of HTML code other than indentation. It is created due to running `./tools/lint --fix`